railties/CHANGELOG.md

```diff
@@ -140,4 +140,8 @@
 
     *Alex Robbin*
 
+*   Rename `railties/bin` to `railties/exe` to match the new Bundler executables convention.
+
+    *Islam Wazery*
+
```

Comment:
New entries should be added to the top of the changelog.

We should probably clarify that in the contributing guide. :confused: